### DISCUSSION TITLE
How to get the last nightly build ?

### DISCUSSION THREAD

**Discussion by ericemc3** [duckdb/duckdb-spatial#216] [General]
I'd like to play with the last dev version, i am on Windows.

**Comment by carlopi** [comment]
`FORCE INSTALL spatial FROM 'http://nightly-extensions.duckdb.org';`
should download and safe the latest development version for any platform.

**Comment by ericemc3** [comment]
Thank you for the prompt reply!

Actually, i'd like to test the new ST_quadkey() function:
`SELECT ST_QuadKey(st_point(11.08, 49.45), 10);`

With DuckDB 0.9.2, after FORCE INSTALL, ST_quadkey is not available:
`Scalar Function with name st_quadkey does not exist!`

With DuckDB 0.9.3 dev, i get a FORCE INSTALL error:
`Failed to download extension "spatial" at URL "http://nightlyextensions.duckdb.org/0ff2c6cb29/windows_amd64/spatial.duckdb_extension.gz" `

**Reply by carlopi** [reply]
I just tried again, with DuckDB v0.9.2:

```
D FORCE INSTALL spatial FROM 'http://nightly-extensions.duckdb.org';
D LOAD spatial;
D SELECT ST_QuadKey(st_point(11.08, 49.45), 10);
```
it should work the same on windows_amd64 platform.

**Comment by ericemc3** [comment]
Thank you, i just tried again, and it works for me now, I guess my previous extension was locked.
